common.mk

```diff
@@ -126,10 +126,20 @@ define mfc_extra_anno_contents
 	}
 ]
 endef
+define sfc_extra_low_transforms_anno_contents
+[
+	{
+		"class": "firrtl.stage.RunFirrtlTransformAnnotation",
+		"transform": "barstools.tapeout.transforms.ExtraLowTransforms"
+	}
+]
+endef
 export mfc_extra_anno_contents
-$(FINAL_ANNO_FILE) $(MFC_EXTRA_ANNO_FILE): $(ANNO_FILE)
+export sfc_extra_low_transforms_anno_contents
+$(EXTRA_ANNO_FILE) $(MFC_EXTRA_ANNO_FILE) $(SFC_EXTRA_ANNO_FILE): $(ANNO_FILE)
```

Comment:
This should be a grouped target `&:` since you are generating multiple files from one